meetings/db.py

```diff
@@ -18,6 +19,16 @@ async def upgrade_v1(conn: Connection) -> None:
          timestamp TEXT NOT NULL,
          sender TEXT NOT NULL,
          message TEXT NOT NULL,
-         tag TEXT DEFAULT NULL
+         tag TEXT DEFAULT NULL,
+         topic TEXT DEFAULT NULL
```

Comment:
Should `topic` be in both v1 and v2 schema? won't that cause an error for a newly-created DB?

Reply:
nope! that is a leftover that i forgot to remove!

Reply:
Cool, I can fix that on merge.